binding.cc

```diff
@@ -474,62 +489,47 @@ struct Database {
 struct PriorityWorker : public BaseWorker {
   PriorityWorker (napi_env env, Database* database, napi_value callback, const char* resourceName)
     : BaseWorker(env, database, callback, resourceName) {
-      database_->IncrementPriorityWork();
+      database_->IncrementPriorityWork(env);
```

Comment:
So basically you pass `env` around all the time instead of hiding it somewhere in a base class?

Reply:
Exactly. This way, anywhere we use it, we know that we safely can.